server/models/user.js

```diff
@@ -46,8 +46,8 @@ module.exports = knex => {
 
   const verify = (username, password) => {
     const matchErrorMsg = 'Username or password do not match'
-
-    knex.select()
+    // return the result to the calling function
+    return knex.select()
```

Comment:
hmm, yeah i missed this. i think i had originally written this function to return a promise and then just `resolve(user)` in the callbacks. but after removing the promise i forgot about the return value. nice catch :)

originally was doing something like:

```
const verify = (username, password) => new Promise((resolve, reject) => {
  knex.select()
    .then([user, isMatch] => {
      if (!isMatch) return reject(matchErrorMsg)
      
      resolve(user)
    })
})
```